package.json

```diff
@@ -18,7 +18,7 @@
     "just-debounce-it": "^1.1.0",
     "name-initials": "^0.1.3",
     "react-native-keyboard-aware-scroll-view": "^0.8.0",
-    "react-native-otp": "^1.0.0",
+    "react-native-otp": "github:l3satwik/react-native-otp#cell-parent",
```

Comment:
This is a temporary solution to support styles (borderX) for <Text /> on iOS for OTPInput.